### PR TITLE
[Workflow Enablement] ARMOCP-525: Use HyperShift cli create HostedCluste with CP on ARM64

### PR DESCRIPTION
**Pull request by LiangquanLi930**

### CODE REVIEW

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-arm64-nightly-aws-ipi-ovn-hypershift-guest-p1-f7

**Comment by bryan-cox** [comment]
/retitle ARMOCP-431: hypershift: guest e2e with CP on ARM64

**Comment by bryan-cox** [comment]
/retitle ARMOCP-525: hypershift: guest e2e with CP on ARM64

**Comment by LiangquanLi930** [comment]
@kuiwang02 Could you help to review this PR? thanks

**Comment by kuiwang02** [comment]
iamok from golang's view.

**Comment by LiangquanLi930** [comment]
@liangxia PTAL this PR, Thanks!

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml [hunk 1]
```diff
@@ -589,6 +589,57 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-ingress-lbtype-nlb
+- as: aws-ipi-ovn-hypershift-guest-p1-f7
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml:592]: When a profile is both covered on amd and arm, migrate half of the tests from amd to arm.
E.g, for a f7 profile which only covers on amd previously, and now covers on both amd and arm, use f14 for both of amd and arm.

Comment (YuLi517) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml:592]: Currently, this is the first ARM profile. Content is Control plane on ARM64, Hosted cluster on AMD64. In 4.14, we may have  Content is Control plane on AMD64, Hosted cluster on ARM64. Then we will consider to use f14 for both profiles.

Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml:592]: Isn't it an earlier agreement that migrate half of the test from amd to arm when adding new profiles for arm ?

Comment (YuLi517) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml:592]: Yes, this is agreement. But currently, on hypershift cluster installation, this is the only profile. Rest of ARM or AMD profiles are using for OCP standard cluster.
In additional, no worry about future profile adding. There won't be much ARM profiles on HyperShift side. 
Based on roadmap, we may have 
AWS:
1. Control Plane: ARM64 + Hosted cluster: AMD64;
2. Control Plane: AMD64 + Hosted cluster: ARM64;
Azure:
1. Control Plane: ARM64 + Hosted cluster: AMD64;
2. Control Plane: AMD64 + Hosted cluster: ARM64;

**Pull request merged**